internal/services/network/ip_group_resource.go

```diff
@@ -47,6 +51,22 @@ func resourceIpGroup() *pluginsdk.Resource {
 
 			"resource_group_name": commonschema.ResourceGroupName(),
 
+			"firewall_ids": {
+				Type:     pluginsdk.TypeSet,
+				Computed: true,
+				Elem: &pluginsdk.Schema{
+					Type: pluginsdk.TypeString,
+				},
+			},
+
+			"firewall_policy_ids": {
```

Comment:
could we add these to the documentation?

Reply:
Hey no problem - sorry was under the impression the docs are auto-generated. Let me find them

Reply:
Done